### PR TITLE
OSDOCS-8034  What's new summary for CLI security groups configuration

### PR DESCRIPTION
**Pull request by laubai**
Do not merge until ROSA CLI 1.2.31 is released (planned Nov 27).

Version(s):
4.14, 4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-8034
https://issues.redhat.com/browse/OSDOCS-8785

Link to docs preview:
https://67456--docspreview.netlify.app/openshift-rosa/latest/rosa_release_notes/rosa-release-notes#rosa-q4-2023_rosa-whats-new

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by laubai** [comment]
@xueli181114 @jmekkatt Minor review for the "What's New" document related to OSDOCS-8037

File: rosa_release_notes/rosa-release-notes.adoc [hunk 1]
```diff
@@ -16,6 +16,8 @@ toc::[]
 [id="rosa-q4-2023_{context}"]
 === Q4 2023
 
+* **Configure custom security groups.** With the release of ROSA CLI (`rosa`) version 1.2.31, administrators can use the use the `rosa create cluster` and `rosa create machinepool` commands to configure a new cluster or a new machine pool with up to 5 additional custom security groups. Additionally, administrators can create a new machine pool with up to 5 additional custom security groups using OpenShift Cluster Manager. This allows administrators to have greater control over access to resources in new clusters and machine pools. For more information, see xref:../rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc#rosa-security-groups_prerequisites[Security groups].
```
Comment (dfitzmau) [inline rosa_release_notes/rosa-release-notes.adoc:19]: 1. Remove extra "use the". 
2. using/by using as per IBM SG
3. "Allow" is a form of anthropomorphism. Could the statement be revised to something such as "By configuring custom security groups, an administrator can have greater..."?

Screenshot from 2023-11-27 12-05-11

Comment (laubai) [reply rosa_release_notes/rosa-release-notes.adoc]: Thanks @dfitzmau, good catch on the double words - going to blame my own butterfingers and unintentional vi commands for that one.

**Comment by xueli181114** [comment]
LGTM

**Pull request merged**